tools/autodeployment/src/docker_events/failCheck.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
```

Comment:
Let's add a comment block to the top of this file explaining what you're doing and why.